src/share/content_renderer.js

```diff
@@ -57,11 +57,18 @@ function getContent(note) {
             }
 
             content = document.body.innerHTML;
+ 
+            if (content.includes("<span class=\"math-tex\">")) {
+                content += "<script src=\"/libraries/katex/katex.min.js\"></script>"
```

Comment:
Use relative instead of absolute URLs since trilium can be running from a subpath.

Reply:
Would that be "../"?